--- a/FLEET.PAS
+++ b/FLEET.PAS
@@ -18,7 +18,6 @@ USES Strg,
      DataCnst,
      Misc,
      Primintr,
-     Orders,
      News;
 
 PROCEDURE AbortFleet(Flt1: IDNumber; GroundID: IDNumber; Report: Boolean);
@@ -213,7 +212,6 @@ PROCEDURE DestroyFleet(FltID: IDNumber);
    VAR
       FltPos: XYCoord;
       FltSta: Empire;
-      Code: OrderStructure;
 
    BEGIN
 	IF FltID.Index IN SetOfActiveFleets THEN
@@ -224,12 +222,6 @@ PROCEDURE DestroyFleet(FltID: IDNumber);
       	FltSta:=Emp;
       	END;
 
-   	IF FleetNextStatement(FltID)>0 THEN
-      	BEGIN
-      	GetFleetCode(FltID,Code);
-      	DisposeOrders(Code);
-      	END;
-
    	Dispose(Universe^.Fleet[FltID.Index]);
 
    	SetOfActiveFleets:=SetOfActiveFleets-[FltID.Index];
@@ -489,133 +481,6 @@ procedure RefuelFleet(FltID,GroundID: IDNumber; Trillum: Resources);
       END;
    END;  { RefuelFleet }
 
-PROCEDURE ExecuteDestCOM(FltID: IDNumber; Loc: Location);
-   BEGIN
-   IF SameXY(Loc.XY,Limbo) THEN
-      GetCoord(Loc.ID,Loc.XY);
-   SetFleetDestination(FltID,Loc.XY);
-   END;  { ExecuteDestCOM }
-
-FUNCTION GetResource(VAR Sh: ShipArray; VAR Cr: CargoArray; Res: ResourceTypes): Word;
-   BEGIN
-   IF Res>trn THEN
-      GetResource:=Cr[Res]
-   ELSE
-      GetResource:=Sh[Res];
-   END;  { GetResource }
-
-PROCEDURE PutResource(VAR Sh: ShipArray; VAR Cr: CargoArray; Res: ResourceTypes; Val: Word);
-   BEGIN
-   IF Res>trn THEN
-      Cr[Res]:=Val
-   ELSE
-      Sh[Res]:=Val;
-   END;  { GetResource }
-
-PROCEDURE ExecuteTransCOM(FltID: IDNumber; Res: ResourceTypes; Trans: Integer);
-   VAR
-      GroundID: IDNumber;
-      FltXY: XYCoord;
-      FltRes,GrnRes: Word;
-      GrnSh,FltSh: ShipArray;
-      GrnCr,FltCr: CargoArray;
-
-   BEGIN
-   GetCoord(FltID,FltXY);
-   GetObject(FltXY,GroundID);
-   IF (NOT SameID(GroundID,EmptyQuadrant)) AND (GetStatus(GroundID)=GetStatus(FltID)) THEN
-      { ASSERT: legal ground exists }
-      BEGIN
-      GetShips(GroundID,GrnSh);
-      GetCargo(GroundID,GrnCr);
-      GetShips(FltID,FltSh);
-      GetCargo(FltID,FltCr);
-
-      FltRes:=GetResource(FltSh,FltCr,Res);
-      GrnRes:=GetResource(GrnSh,GrnCr,Res);
-      
-      IF Trans>0 THEN
-         { ASSERT: fleet is picking up stuff }
-         BEGIN
-         Trans:=LesserInt(Trans,GrnRes);
-         Trans:=LesserInt(Trans,MaxResources-FltRes);
-         IF Res IN [men..tri] THEN
-            Trans:=LesserInt(FleetCargoSpace(FltSh,FltCr)*LongInt(CargoSpace[Res]),Trans);
-         END
-      ELSE
-         { ASSERT: fleet is dropping stuff off }
-         BEGIN
-         Trans:=-Trans;
-         Trans:=LesserInt(Trans,FltRes);
-         Trans:=LesserInt(Trans,MaxResources-GrnRes);
-         Trans:=-Trans;
-         END;
-
-      PutResource(FltSh,FltCr,Res,FltRes+Trans);
-      PutResource(GrnSh,GrnCr,Res,GrnRes-Trans);
-
-      BalanceFleet(FltSh,FltCr);
-      ChangeCompositionOfFleet(FltID,GroundID,FltSh,FltCr,GrnSh,GrnCr);
-      END;
-   END;  { ExecuteTransCOM }
-
-PROCEDURE ExecuteFleetOrders(Emp: Empire; FltID: IDNumber);
-{ ExecuteFleetOrders: ----------------------------------------------------------
-   This procedure should be called after a fleet has reached its destination
-   but before a player takes his/her turn.
------------------------------------------------------------------------------- }
-   VAR
-      Code: OrderStructure;
-      FleetDestroyed,IgnoreRepeat: Boolean;
-      LastCommand,Com: Word;
-      Command: CommandRecord;
-
-   BEGIN
-   Com:=FleetNextStatement(FltID);
-   IgnoreRepeat:=False;
-	FleetDestroyed:=False;
-   IF Com<>0 THEN
-      { ASSERT: Fleet has orders }
-      BEGIN
-      GetFleetCode(FltID,Code);
-      LastCommand:=NumberOfCommands(Code);
-      REPEAT
-         GetCommandRecord(Code,Com,Command);
-         CASE Command.Typ OF
-            DestCOM: ExecuteDestCOM(FltID,Command.Loc);
-            TransCOM: ExecuteTransCOM(FltID,Command.Res,Command.Trns);
-            END; { case }
-
-         IF FltID.Index IN SetOfActiveFleets THEN
-            { ASSERT: Fleet still exists. }
-            BEGIN
-            IF (Command.Typ=RepeatCOM) AND (NOT IgnoreRepeat) THEN
-               BEGIN
-               Com:=1;
-               IgnoreRepeat:=True;
-               END
-            ELSE
-               BEGIN
-               IF Com<LastCommand THEN
-                  Inc(Com)
-               ELSE
-                  BEGIN
-                  Com:=0;
-                  DisposeOrders(Code);
-                  SetFleetCode(FltID,Code);
-                  END;
-               END;
-            END
-         ELSE
-            FleetDestroyed:=True;
-
-      UNTIL (Command.Typ IN [DestCOM,WaitCOM]) OR (Com=0) OR (FleetDestroyed);
-
-      IF NOT FleetDestroyed THEN
-         SetFleetNextStatement(FltID,Com);
-      END;
-   END;  { ExecuteFleetOrders }
-
 FUNCTION InRangeOfDisrupter(Emp: Empire; Pos: XYCoord; VAR DisruptBy: Empire): Boolean;
    VAR
       i: Word;
@@ -852,10 +717,6 @@ PROCEDURE UpdateFleet(FltID: IDNumber);
          END;
       END;
 
-   IF (NOT FltDestroyed) AND (GetFleetStatus(FltID)=FReady) THEN
-      BEGIN
-      ExecuteFleetOrders(Emp,FltID);
-      END;
    END;  { UpdateFleet }
 
 PROCEDURE UpdateAllFleets(Player,NextPlayer: Empire);
